__fixtures__/templates/modular-template-app/README.md

```diff
@@ -0,0 +1,39 @@
+# `PackageName__`
+
+This is a [Modular App](https://modular.js.org/package-types/app)
+
+## Build
+
+To [build](https://modular.js.org/commands/build) your app for deployment, run:
+
+```bash
+modular build PackageName__
+```
+
+The resulting output is an optimized site that can be served statically. All
+code (files in `src` plus external dependencies required in the code) is bundled
+in a single blob of code that can be split in different files.
+
+## Start
+
+To run your app locally on a development server, run
+[start](https://modular.js.org/commands/start):
+
+```bash
+modular start PackageName__
+```
+
+This causes a developer server to run on port 3000, serving the app with an
```

Comment:
*very* nit-picky but _This starts a developer server on port 3000_ 
might sound a bit cleaner here

Reply:
Done. I also replaced all the fixture READMEs with a placeholder, since we don't need to necessarily sync them with the real READMEs.